quickstarts/post-configure-example/README.md

```diff
@@ -0,0 +1,200 @@
+
+
+Note that, this procedure can be applied to any kid of post configure actions.
+
+## Preparing the needed files
+
+Before moving forward with the deployments, let's prepare the needed files.
+You need to have the following files:
+
+- [add-users.cli](add-users.cli): JBoss CLI Batch script, your script must be added between the commands below:
+  - ```bash
+    embed-server --std-out=echo  --server-config=standalone-openshift.xml
+    batch
+
+    <your jboss-cli commands>
+
+    run-batch
+    quit
+    ```
+    
+- [delayedpostconfigure.sh](delayedpostconfigure.sh): This empty file is needed until the following jira is not fixed:
+  https://issues.redhat.com/browse/RHPAM-3665
+
+- [postconfigure.sh](postconfigure.sh): The script that will actually call the JBoss-CLI script.
+
+
+With these 3 files in place, the next step will be creating a config-map and mount it under `/opt/eap/extensions` directory
+on the target container image(s).
+
+```bash
+$ oc create configmap postconfigure \
+    --from-file=add-users.cli=add-users.cli \
+    --from-file=delayedpostconfigure.sh=delayedpostconfigure.sh \
+    --from-file=postconfigure.sh=postconfigure.sh
+configmap/postconfigure created
+```
+
+If you inspect the created config-map, the result would be something like:
+
+```bash
+$ oc describe cm postconfigure
+Name:         postconfigure
+Namespace:    add-users-example
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+add-users.cli:
+----
+embed-server --std-out=echo  --server-config=standalone-openshift.xml
+batch
+
+/subsystem=elytron/filesystem-realm=KieFsRealm:add-identity(identity=user1)
+/subsystem=elytron/filesystem-realm=KieFsRealm:set-password(identity=user1, clear={password="pass123*"})
+/subsystem=elytron/filesystem-realm=KieFsRealm:add-identity-attribute(identity=user1, name=role, value=["kie-server","rest-all","admin","kiemgmt","Administrators","user"])
+
+/subsystem=elytron/filesystem-realm=KieFsRealm:add-identity(identity=user2)
+/subsystem=elytron/filesystem-realm=KieFsRealm:set-password(identity=user2, clear={password="pass123*"})
+/subsystem=elytron/filesystem-realm=KieFsRealm:add-identity-attribute(identity=user2, name=role, value=["kie-server","rest-all","admin","kiemgmt"])
+
+run-batch
+quit
+
+delayedpostconfigure.sh:
+----
+This empty file is needed until the following jira is not fixed:
+https://issues.redhat.com/browse/RHPAM-3665
+
+postconfigure.sh:
+----
+echo "******  RUNNING ADDITIONAL CONFIGURATIONS WITH JBOSS-CLI - ADDING EXTRA ELYTRON USERS TO KIE FS REALM **********"
+echo "trying to execute /opt/eap/bin/jboss-cli.sh --file=/opt/eap/extensions/add-users.cli"
+/opt/eap/bin/jboss-cli.sh --file=/opt/eap/extensions/add-users.cli
+echo "END - users added"
+
+Events:  <none>
+```
+
+
+With the config-map created, you can now proceed with the container configuration.
+In this example there is only one KIE Server running. The steps will be described by using the `operator` and 
+`application templates`.
```

Comment:
"The steps will be described by using the `operator` and  `application templates`." I think this is unclear. Suggestion: "the steps are different depending on whether you use the operator or application  templates to deploy the product".